app/main.cpp

```diff
@@ -57,6 +60,7 @@ int main(int argc, char *argv[])
     app.setApplicationName("CuteMarkEd");
     app.setApplicationDisplayName("CuteMarkEd");
     app.setApplicationVersion("0.11.3");
+    QResource::registerResource(QDir::currentPath()+"/math_reveal.rcc");
```

Comment:
looks dangerous, especially on Linux. program will try to find "math_reveal.rcc" in $PWD, this is unacceptable in most cases
file would be found only if program will be run from folder where it located
this will not work for Windows too (when double clicking on file), except case when program is started by double clicking _.exe_ and then _File->Open_ called